EST.MIT.InvoiceImporter.Function/Functions/UploadFunctions.cs

```diff
@@ -39,4 +41,27 @@ public UploadFunctions(IAzureTableService azureTableService)
             return new StatusCodeResult((int)HttpStatusCode.InternalServerError);
         }
     }
+
+    [FunctionName("GetUploadedFile")]
+    public async Task<IActionResult> GetUploadedFile(
+      [HttpTrigger(AuthorizationLevel.Function, "get", Route = "UploadedFile/{Ref}")] HttpRequest req,
```

Comment:
How will this http endpoint be called? You're defining Function-level auth which means you'll need to included the SAS token in the http call. An you'll presumably need to store the SAS token in Key Vault. Or if this call comes from the invoice API say, could we include the AAD bearer token and validate it as part of the Function or middleware? It may be that we've not got to that point yet (since I don;t think we have web login setup yet) but worth thinking about